### ISSUE TITLE
site.baseurl not set for the demo site

### ISSUE THREAD

**Issue by atikhono** [poole/poole#86]
Hi Mark, browsing Related Posts at the Poole demo site causes a 'can't connect to the server' error. Looks like it happens because site.baseurl is set to ''.

**Comment by nicolasmccurdy** [comment]
I'm seeing the same issue. It seems like this is related to #26.

**Comment by maurogestoso** [comment]
I've experienced the same problem. Doing some research I've found [this very helpful article](https://byparker.com/blog/2014/clearing-up-confusion-around-baseurl/) on how to use baseurl properly. 

If you are using a project page, baseurl should be "/project-name". If it's your main github page, then leave it as "".

The problem is in the post layout, on the href of the link to each related post. The "/" between {{ site.baseurl }} and {{ post.url }} should be deleted. If you check index.html, the same link is used on the heading of each post and the "/" is not there. Works for me...
